### DISCUSSION TITLE
What would you like to see in Botonic v1.0?

### DISCUSSION THREAD

**Discussion by ericmarcos** [hubtype/botonic#1371] [General]
Hi all!

We're preparing the roadmap for our long awaited 1.0 release!
We'd like to get feedback from the community about the features you would like to see in this version.
Some features we're already working on:
* Self-hosted version
* Named Entity Recognition
* A new Botonic NLP data input format
* Migrating to Typescript

Let us know your thoughts: is there anything missing? Are these features relevant to your project?
Thanks!

Botonic Team

**Comment by zscgeek** [comment]
Hi - any update on the timing of 1.0?
